docs/configuration.md

```diff
@@ -168,6 +168,8 @@ Flags:
       --feature-set string                       Minimum feature set to enable by default: alpha, beta, or stable. Warning: modify at own risk. (default "stable")
       --feature-set-disable strings              Comma-separated list of features to disable, overriding the default minimum feature set.
       --feature-set-enable strings               Comma-separated list of features to enable, overriding the default minimum feature set.
+      --graffiti strings                         Comma separated list of graffiti strings to include in block proposals. Maximum 32 ASCII characters per graffiti.
```

Comment:
See other "Comma-separated" strings just above this one. Maybe need to add dash as well?